Web/starter_project_barbies/components/about/ProjectCard.tsx

```diff
@@ -0,0 +1,49 @@
+import { ProjectCardProps } from '@/types/AboutTypes'
+import Image from 'next/image'
+
+const ProjectCard = ({ project, isEven }: ProjectCardProps) => {
```

Comment:
Let's define props here above the component so it's more clear. I think it would be best to do it for all the components whose props are define on types folder